### PR TITLE
Use better instruction for cors settings with MetaMask

### PR DESCRIPTION
**Pull request by frankisawesome**
Signed-off-by: Frank Li <b439988l@gmail.com>

## Pull request checklist

Use the following list to make sure your PR fits the Besu doc quality standard.

### Before creating the pull request

Make sure that:

- [x] [all commits in this PR are signed off for the DCO](https://wiki.hyperledger.org/display/BESU/DCO).
- [x] you read the [contribution guidelines](https://wiki.hyperledger.org/display/BESU/Contributing+to+documentation).
- [x] you have [tested your changes locally](https://wiki.hyperledger.org/display/BESU/MkDocs+And+Markdown+Guide#MkDocsAndMarkdownGuide-PreviewTheDocumentation) before submitting them to the community for review.

### After creating your pull request and tests finished

Make sure that:

- [ ] you fixed all the issues raised by the tests, if any.
- [ ] you verified the rendering of your changes on [ReadTheDocs.org PR preview](https://wiki.hyperledger.org/display/BESU/MkDocs+And+Markdown+Guide#MkDocsAndMarkdownGuide-PreviewwithReadTheDocs)
  and updated the testing link (see [Testing](#testing)).

## Describe the change

Include the MetaMask domain needed to whitelist in the cors option and provide a clearer message.

Question coming from rocket chat https://chat.hyperledger.org/channel/besu/thread/xtJMFeHfwb4D7rg9x

### For content changes

- [ ] Doc content
- [ ] Doc pages organisation

### For tools changes

- [ ] CircleCI workflow
- [ ] Build and QA tools (lint, vale,…)
- [ ] MkDocs templates
- [ ] MkDocs configuration
- [ ] Python dependencies
- [ ] Node dependencies and JavaScript
- [ ] ReadTheDocs configuration
- [ ] GitHub integration

### CODE REVIEW

File: docs/Reference/CLI/CLI-Syntax.md [hunk 1]
```diff
@@ -2632,9 +2632,10 @@ with your Besu node.
 
 !!!note
 
-    To run a local Besu node as a backend for MetaMask and use MetaMask anywhere, set
-    `--rpc-http-cors-origins` to `"all"` or `"*"`. To allow a specific domain to use MetaMask with
-    the Besu node, set `--rpc-http-cors-origins` to the client domain.
+    To run a local Besu node as with MetaMask, set `--rpc-http-cors-origins` to
```
Comment (bgravenorst) [inline docs/Reference/CLI/CLI-Syntax.md:2635]: "as with" -> "for"?

File: docs/Reference/CLI/CLI-Syntax.md [hunk 2]
```diff
@@ -2632,9 +2632,10 @@ with your Besu node.
 
 !!!note
 
-    To run a local Besu node as a backend for MetaMask and use MetaMask anywhere, set
-    `--rpc-http-cors-origins` to `"all"` or `"*"`. To allow a specific domain to use MetaMask with
-    the Besu node, set `--rpc-http-cors-origins` to the client domain.
+    To run a local Besu node as with MetaMask, set `--rpc-http-cors-origins` to 
+    `chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn`.
+    To allow a specific domain to use MetaMask with the Besu node, set `--rpc-http-cors-origins` to 
+    the client domain. Remember to also include the dApp domain MetaMask interacts with.
```
Comment (bgravenorst) [inline docs/Reference/CLI/CLI-Syntax.md:2638]: We use "dapp" as a standard in the docs.

**Review by bgravenorst — APPROVED** [review]
Minor suggestions. Can be merged once fixed.

File: docs/Reference/CLI/CLI-Syntax.md (hunk 2 above)
Comment (macfarla) [reply docs/Reference/CLI/CLI-Syntax.md]: can we provide an example? eg with client domain and dapp domain?

File: docs/Reference/CLI/CLI-Syntax.md (hunk 1 above)
Comment (frankisawesome) [reply docs/Reference/CLI/CLI-Syntax.md:2635]: Haha good catch! thank you

**Review by macfarla — APPROVED** [review]

File: docs/Reference/CLI/CLI-Syntax.md [hunk 3]
```diff
@@ -2632,9 +2632,12 @@ with your Besu node.
...
 !!!note
 
-    To run a local Besu node as a backend for MetaMask and use MetaMask anywhere, set
-    `--rpc-http-cors-origins` to `"all"` or `"*"`. To allow a specific domain to use MetaMask with
-    the Besu node, set `--rpc-http-cors-origins` to the client domain.
+    To run a local Besu node with MetaMask, set `--rpc-http-cors-origins` to 
+    `chrome-extension://nkbihfbeogaeaoehlefnkodbefgpgknn`.
+    
+    Remember to also include the dapp domain MetaMask interacts with, for example if your app is deployed 
+    on remix and you're using metamask to interact with the contract the config option should be
```
Comment (macfarla) [inline docs/Reference/CLI/CLI-Syntax.md:2639]: nit: MetaMask

Comment (macfarla) [reply docs/Reference/CLI/CLI-Syntax.md:2639]: and maybe Remix

**Review by macfarla — APPROVED** [review]

**Pull request merged**